specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/AnomalyDetector.json

```diff
@@ -0,0 +1,504 @@
+        "description": "Evaluate change point score of every series point",
+        "operationId": "DetectChangePoint",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Time series points and granularity is needed. Advanced model parameters can also be set in the request if needed.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/ChangePointDetectResponse"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/AnomalyDetectorError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Detect change point example": {
+            "$ref": "./examples/ChangePointDetect.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnomalyDetectorError": {
+      "type": "object",
+      "description": "Error information returned by the API.",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "enum": [
+            "InvalidCustomInterval",
+            "BadArgument",
+            "InvalidGranularity",
+            "InvalidPeriod",
+            "InvalidModelArgument",
+            "InvalidSeries",
+            "InvalidJsonFormat",
+            "RequiredGranularity",
+            "RequiredSeries"
+          ],
+          "x-ms-enum": {
+            "name": "AnomalyDetectorErrorCodes",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "description": "A message explaining the error reported by the service.",
+          "type": "string"
+        }
+      }
+    },
+    "TimeGranularity": {
+      "type": "string",
+      "description": "Optional argument, can be one of yearly, monthly, weekly, daily, hourly, minutely, secondly, microsecond or none. If granularity is not present, it will be none by default. If granularity is none, the timestamp property in time series point can be absent.",
+      "x-nullable": false,
+      "x-ms-enum": {
+        "name": "TimeGranularity",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "yearly"
+          },
+          {
+            "value": "monthly"
+          },
+          {
+            "value": "weekly"
+          },
+          {
+            "value": "daily"
+          },
+          {
+            "value": "hourly"
+          },
+          {
+            "name": "perMinute",
+            "value": "minutely"
```

Comment:
>minutely [](http://example.com/codeflow?start=22&length=8)

I am not a native speaker, but this seems not to mean every minute.

Reply:
This is from the existing API. If it is not a good option, maybe we need to discuss and fix this problem in a new thread.